src/Orc.NuGetExplorer.Xaml/ViewModels/PageItemViewModel.cs

```diff
@@ -71,6 +75,16 @@ private void CheckItemExecute(MouseButtonEventArgs parameter)
 
         protected override Task InitializeAsync()
         {
+            // Handle only on Browse page
+            if (ParentViewModel is ExplorerPageViewModel explorerParent)
+            {
+                if (explorerParent.Page.Parameters.Tab == ExplorerTab.Browse)
+                {
+                    Package.StatusChanged += OnPackageStatusChanged;
+                    _nugetSettings.PropertyChanged += OnNuGetSettingsChanged;
+                }
```

Comment:
move into separate method `InitializeExplorerPage()`  or any other choose appropriate method name

Reply:
Not a initialization page, i just need to subscribe here to events _in one specific case_
Will it create any value/more readability if we will have a method like `HandlePackageEnlistingStateIfBrowsePageActive`
I guess having a methods which aggregate very specific logic can make the class readability worse, why leaving explanation comment directly gaves idea what happens here.
It's probably this place should be refactored (and moved out) in separate class/service, will leave as-is